boards/arm/arduino_nicla_sense_me/board.cmake

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: Apache-2.0
+
+set(OPENOCD_NRF5_SUBFAMILY "nrf52")
```

Comment:
Why is this here if you are not including openocd support? Please either remove this line or add openocd support to the board.

Reply:
I probably thought it had some effect on the pyocd... I'll just remove that line since it works without it 👍

Reply:
Regarding support for openocd, I just can't get anything other than pyocd working.